pkg/asset/images.go

```diff
@@ -3,11 +3,11 @@ package asset
 // DefaultImages are the defualt images bootkube components use.
 var DefaultImages = ImageVersions{
 	Etcd:            "quay.io/coreos/etcd:v3.3.12",
-	Flannel:         "quay.io/coreos/flannel:v0.10.0-amd64",
+	Flannel:         "quay.io/coreos/flannel:v0.11.0-amd64",
 	FlannelCNI:      "quay.io/coreos/flannel-cni:v0.3.0",
 	Calico:          "quay.io/calico/node:v3.0.3",
 	CalicoCNI:       "quay.io/calico/cni:v2.0.0",
-	CoreDNS:         "k8s.gcr.io/coredns:1.2.6",
-	Hyperkube:       "k8s.gcr.io/hyperkube:v1.13.3",
+	CoreDNS:         "k8s.gcr.io/coredns:1.6.2",
+	Hyperkube:       "k8s.gcr.io/hyperkube:v1.16.2",
```

Comment:
I feel like the flannel and CoreDNS changes should be in a separate PR since they are not strictly required in the upgrade to  Kubernetes 1.16.